cmd/sandbox-ce/compose/compose.go

```diff
@@ -2,16 +2,135 @@ package compose
 
 import (
 	"context"
+	"fmt"
+	"io"
+	"net/http"
 	"os"
 	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/pkg/errors"
 )
 
-func Run(ctx context.Context, arg ...string) error {
-	cmd := exec.CommandContext(ctx, "docker-compose", arg...)
+// composeContainerPath is the url docker-compose is downloaded from in case
+// that it's not already present in the system
+const composeContainerPath = "https://github.com/docker/compose/releases/download/1.24.0/run.sh"
```

Comment:
If this is a bash, we should have some checks and avoid to download and run it if the os is windows.

Reply:
Addressed [here](https://github.com/smacker/superset-compose/pull/4/commits/b27087441b3db77b98c4b0a648122006858b582e).